openassessment/xblock/static/js/spec/lms/oa_datefactory.js

```diff
@@ -0,0 +1,84 @@
+/**
+Tests for OA dateutil factory.
+**/
+
+describe('OpenAssessment.DateTimeFactory', function() {
+
+    beforeEach(function() {
+        // Load the DOM fixture
+        loadFixtures('oa_response_date.html');
+
+    });
+    describe('transform', function() {
+        it('has the correct HTML elements', function() {
+            var timeElement = $('.step__title').get(0);
+            var datetimeFactory = new OpenAssessment.DateTimeFactory(timeElement);
+            datetimeFactory.report();
+            $('.ora-datetime', timeElement).each(function() {
```

Comment:
@andy-armstrong I absolutely agree with you. In practice, the platform has loaded the dependencies well before this is called, however if we decide in the future to deepen the require dependency in edx-ora, there might be some refactoring that can be done.
This seemed to be the best solution in this interim state, and if we do move forward in the future, I'm confident that it'll continue to work as is.